### PR TITLE
Add e2e test for GitOpsDeploymentSyncRun API

### PR DESCRIPTION
**Pull request by chetan-rns**
#### Description:
- e2e test for GitOpsDeploymentSyncRun API

#### Link to JIRA Story (if applicable):

https://issues.redhat.com/browse/GITOPSRVCE-153

### CODE REVIEW

**Comment by chetan-rns** [comment]
/hold

Will remove hold after #279 is merged

**Review by jgwest — APPROVED** [review]
Great stuff, thanks @chetan-rns, and thanks for your patience in getting this in!

**Pull request merged**
